authors/apps/profiles/views.py

```diff
@@ -0,0 +1,69 @@
+from rest_framework import status
+from rest_framework.generics import RetrieveUpdateAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+from django.http import Http404
```

Comment:
Hey @alex just wondering why you would import`Http404`, and yet you also have `status` imported l thought it dos the same thing  [Initially] if you have this `from rest_framework import status` even this `HTTP_404_NOT_FOUND` is handled. You could please look at this resource[https://www.django-rest-framework.org/api-guide/status-codes/](url) Unless it is doing something that l don't know about. 
l stand to be collected if am wrong
Thanks

Reply:
@jennizalwango kindly not that this has been worked on.